CHANGELOG for model updates published around: Mon Aug 31 18:31:17 UTC 2020 (#2987)
aws-amplify/aws-sdk-ios: CHANGELOG.md

---

MOVE THIS SECTION TO THE CORRECT LOCATION

# AWS Mobile SDK for iOS CHANGELOG

## Unreleased

### Note for Xcode 12 support
- All SDKs have been updated to support minimum iOS version 9.0, dropping support for iOS 8.0 in this release. Xcode 12 has dropped support for iOS 8 and requires upgrading to this version to continue building on Xcode 12. ([PR #2981](https://github.com/aws-amplify/aws-sdk-ios/pull/2981))

### Misc. Updates
- Model updates for the following services.

>   - **Breaking Change** Amazon EC2 ([PR #2986](https://github.com/aws-amplify/aws-sdk-ios/pull/2986))

Reviewer: ```suggestion
  - **Breaking Change** Amazon EC2 - reverted a change that was mistakenly released in the previous model update ([PR #2986](https://github.com/aws-amplify/aws-sdk-ios/pull/2986))
```

>   - **Breaking Change** Amazon Elastic Load Balancing (ELB) ([PR #2951](https://github.com/aws-amplify/aws-sdk-ios/pull/2951))

Reviewer: ```suggestion
  - **Breaking Change** Amazon Elastic Load Balancing - updated to API version `2015-12-01` ([PR #2951](https://github.com/aws-amplify/aws-sdk-ios/pull/2951))
```

## Unreleased

### Note for Xcode 12 support

> - All SDKs have been updated to support minimum iOS version 9.0, dropping support for iOS 8.0 in this release. Xcode 12 has dropped support for iOS 8 and requires upgrading to this version to continue building on Xcode 12. ([PR #2981](https://github.com/aws-amplify/aws-sdk-ios/pull/2981))

Reviewer 2: s/upgrading to this version/setting your app's minimum supported version to 9.0 or greater/

Reviewer: ```suggestion
- All SDKs have been updated to support minimum iOS version 9.0, dropping support for iOS 8.0 in this release. Xcode 12 has dropped support for iOS 8 and requires setting your app's minimum supported version to 9.0 or greater to continue building on Xcode 12. ([PR #2981](https://github.com/aws-amplify/aws-sdk-ios/pull/2981))
```

### Misc. Updates
- Model updates for the following services.
  - **Breaking Change** Amazon EC2 ([PR #2986](https://github.com/aws-amplify/aws-sdk-ios/pull/2986))

>   - **Breaking Change** Amazon Elastic Load Balancing (ELB) ([PR #2951](https://github.com/aws-amplify/aws-sdk-ios/pull/2951))

Reviewer 2: Let's add a bit more context:

  - **Breaking Change** Amazon EC2 - reverted a change that was mistakenly released in the previous model update ([PR #2986](https://github.com/aws-amplify/aws-sdk-ios/pull/2986))
  - **Breaking Change** Amazon Elastic Load Balancing - updated to API version `2015-12-01` ([PR #2951](https://github.com/aws-amplify/aws-sdk-ios/pull/2951))
